<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="fileLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>591</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>571</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portField">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
